scripts/error_codes.py

```diff
@@ -213,32 +239,34 @@ def main(argv):
         ["libevmasm", "liblangutil", "libsolc", "libsolidity", "libsolutil", "libyul", "solc"],
         [".h", ".cpp"]
     )
-    used_ids = get_used_ids(source_file_names)
+    source_id_to_file_names = find_ids_in_source_files(source_file_names)
 
     ok = True
-    for id in sorted(used_ids):
+    for id in sorted(source_id_to_file_names):
```

Comment:
Don't use `id` because you are shadowing the builtin symbol `id(obj)` to get the id of an object, maybe use `source_id` instead to avoid name conflicts?

Reply:
p.s.: the same probably applies to your use of the `next` variable, overshadowing the builtin, yielding confusion :)

Reply:
I renamed `next` to `next_id`.

I also started to change `id` to `source_id` or alike, but it does not feel quite right. `id` is used locally, where the context (if it is a source id, or test id) does not matter much. Consider `print_ids` for instance. Renaming would look bulky.

 I agree that shadowing is not nice, but since `id` is a local variable with small scope, I hope it is acceptable.